gan/gan_deepspeed_run.sh

```diff
@@ -0,0 +1 @@
+deepspeed --num_gpus 16 gan_deepspeed_train.py --dataset celeba --cuda --deepspeed_config gan_deepspeed_config.json --tensorboard_path './runs/deepspeed'

```

Comment:
maybe "--num_gpus 16" is not needed, to use all gpus in user's machine?

Reply:
Sure!